packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -286,7 +292,7 @@ class Checkout extends Component<
         }
 
         return (
-            <div className={classNames({ 'is-embedded': isEmbedded() })}>
+            <div className={classNames({ 'is-embedded': isEmbedded(), 'remove_checkout_step_numbers': isHidingStepNumbers })}>
```

Comment:
```suggestion
            <div className={classNames({ 'is-embedded': isEmbedded(), 'remove-checkout-step-numbers': isHidingStepNumbers })}>
```

Reply:
Also should we use `remove-checkout-step-numbers` ?

Reply:
I believe we will remove this class name this or next month, so it is only there to support the experiment's roll-out.
Just want to confirm, are you suggesting changing a name or not to use this approach?

Reply:
Just changing the name.

Reply:
What name(s) do you have in mind?

Reply:
Since it is most likely for us to have a toggle on this, maybe `is-hiding-step-numbers`?